src/main/scala/com/typesafe/scalalogging/Logger.scala

```diff
@@ -93,7 +93,7 @@ object Logger {
 }
 
 /**
- * Implementation for a performant logger based on macros and an underlying `org.slf4j.Logger`.
+ * Implementation of a fast logger based on macros and an underlying `org.slf4j.Logger`.
```

Comment:
can you make this change to the README as well?

Reply:
done